storyStarterGame.js

```diff
@@ -0,0 +1,470 @@
+         } else if (arr1[afterLife] === "rest peacefully") {
+             arr1.splice(afterLife,1)
+             restPeacefully()
+         }
+     } else if (arr1[i] === "avenge death" && arr1[i2] === "rest peacefully") {
+         let afterLife = readLineSync.keyInSelect(arr1, "Do you want them to avenge their death or rest peacefully? ", {cancel: "Must choose above options"})
+         if (arr1[afterLife] === "rest peacefully") {
+             arr1.splice(afterLife,1)
+             restPeacefully()
+         } else if (arr1[afterLife] === "avenge death") {
+             arr1.splice(afterLife, 1)
+             avengeDeath()
+         }
+     }
+} else if (arr1.length=== 0) {
+    replayTheGame()
+}
+}
+
+
+
+ // avengeDeath 
+ const avengeDeath = (charMainName) => {
+console.log(`${charMainName} could not just rest! They want to find out who killed them.`)
+console.log (`Now after endless searching, finally ${charMainName} has found the killer and its their own brother!!!`)
+console.log(`You must choose for what ${charMainName}`)
+let avengerAnswer = readLineSync.question("Do you want to kill your character's brother or do you want to investegate more on the brother?")
+
+if (avengerAnswer === "investegate more") {
+    investegateTheBrother();
+} else {
+} (avengerAnswer === "kill the brother") 
+    console.log(`Your character have killed their own brother. However! \nThey have gain nothing but more regret because ${charMainName} did not investegate more! \nIf they had, ${charMainName} would have found out their brother was trying to save them from fully turning into a zombie! \nWhen ${charMainName}'s brother found them it was far too late on the outside to save them so he did what he thought was a good choice`)
+    campOutArrOptions(); 
+}
+
+
+ // investagateTheBrother
+ const investegateTheBrother = () => {
+     console.log("Wow our brother carried such a painful duty for the country upon his shoulder")
+     console.log("Now listen I provided you with golden materials, however you must now write the story on your own! ")
+     replayTheGame();
+ }
+ 
+
+ //hauntCriminals 
+ const hauntCriminals = () => {
+
+ }
+
+ //restPeacefully
+ const restPeacefully = () => {
+
+ }
+
+ //campOutArrOptions
+ const campOutArrOptions = () => {
+
+ }
+// walkTheWoodsAlone
+const walkTheWoodsAlone = () => {
+    let charMainName = readLineSync.question("What's your main character's name?")
+    console.log(`${charMainName} is walking through the dark woods and they hit a path of three.You have to select the path that ${charMainName} will take.`)
+    arr3 = ["left", "middle", "right"]
+   let woodsPaths = readLineSync.keyInSelect (arr3)
+   let youchoose = arr3[woodsPaths] 
+   for (i = 0; i < arr3.length; i++)
+    if (youchoose === "left") {
+    leftPath ();
+} else if (youchoose === "middle") {
+    middlePath();
+} else if (youchoose === "right") {
+    rightPath();
+} else if (youchoose === undefined){
+    leaveGame();
+}
```

Comment:
Please make sure that all of your code is formatted correctly before submitting.

Your for-loop is not correctly written, it is missing the curly brackets and arr3 was also never defined as a `let` or `const` variable